tests/dummy/app/templates/infinity-scrollable.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="list" style="height: 500px; overflow: scroll;">
+  <ul>
+    {{#each model as |val|}}
+    <div class="infinity-svg">
```

Comment:
Indent?  Only to be consistent with the next file where the `each` block is indented